docs/introduction-to-ai.md

```diff
@@ -11,7 +11,19 @@ This chapter aims to introduce AI as the concept itself, its context along histo
 
 ## What is *Artificial Intelligence*?
 
-`TODO.`
+At this moment in time, Artificial Intelligence doesn't have an exact description, even when is a field with a relative long history. The main problem when defining Artificial Intelligence is that it implies that there is a Natural Ingelligence, so, when talking about humans we can think we have this chracteristic because we can manipulate our reality, but animals can do that too, in a different and more passive way, but they do.
```

Comment:
- "(...) even when ***it*** is a field with a relative long history."
- "(...) we can think we have this ***characteristic*** (...)"